fabfile/__init__.py

```diff
@@ -0,0 +1,20 @@
+from fabric.api import env, task, execute, runs_once
+
+env.roledefs = {
+    'master': ['localhost'],
+    'workers': [ip.strip() for ip in open('worker-instances')],
+}
+env.roles = ['master', 'workers']
+env.forward_agent = True # So remote machines can checkout private git repos
+
+import config # settings, but no tasks
+import pg  # tasks which control postgres
+import add  # tasks which add things to existing clusters
+import setup  # tasks which create clusters with preset settings
+import use  # tasks which change some configuration future tasks read
+import run
+
+@task(default=True)
+@runs_once
+def main():
```

Comment:
I did this because the `create-stack.sh` script runs `fab` automatically when it can and I didn't want to have to hardcode the name of the task to run, in case the appropriate task changes in the future.

I'm a little worried about maintaining all this documentation. If I follow all your comments I'll have to document three places:
- The docstrings of the tasks, to make sure `fab --list` works.
- The README, to list all of the tasks and what they do again.
- The default `fab` task.

Keeping all of these in sync will be a bit tedious.